docs/hub/model-release-checklist.md

```diff
@@ -1,102 +1,134 @@
+3. **Usage Examples**:
+   Provide clear, copy-and-run code snippets for inference, fine-tuning, or other common tasks. Keep edits needed by users to a minimum.
 
-Bonus: You can showcase your model capabilities by placing a well structured `notebook.ipynb` in the model repository. This would allow users to directly open your notebook and run it in [Google Colab and Kaggle Notebooks](https://huggingface.co/docs/hub/en/notebooks) directly.
+   *Bonus*: Add a well-structured `notebook.ipynb` in the repo so users can open it in [Google Colab and Kaggle Notebooks](https://huggingface.co/docs/hub/en/notebooks) directly.
 
-4. **Technical Specifications**: Include information about training parameters, hardware requirements, and any other technical details that would help users understand how to effectively use your model.
+   | ![colab and kaggle button](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/release-checklist/colab-kaggle.png) |
+   | :--: |
+   | Google and Kaggle Usage Buttons |
 
-5. **Performance Metrics**: Share comprehensive benchmarks and evaluation results. Include both quantitative metrics and qualitative examples to give users a complete picture of your model's capabilities and limitations.
+4. **Technical Specifications**:
+   Include training parameters, hardware needs, and other details that help users run the model effectively.
 
-6. **Limitations and Biases**: Transparently document any known limitations, biases, or ethical considerations associated with your model. This helps users make informed decisions about whether and how to use your model.
+5. **Performance Metrics**:
+   Share benchmarks and evaluation results. Include quantitative metrics and qualitative examples to show strengths and limitations.
 
-### Enhancing Model Discoverability and Usability
+6. **Limitations and Biases**:
+   Document known limitations, biases, and ethical considerations so users can make informed choices.
 
-To maximize your model's reach and usability:
+To make the process more seamless, click **Import model card template** to pre-fill the `README.md`s with placeholders.
 
-1. **Library Integration**: If possible, add support for one of the many [libraries integrated with the Hugging Face Hub](https://huggingface.co/docs/hub/models-libraries) (such as Transformers or Diffusers). This integration significantly increases your model's accessibility and provides users with code snippets for working with your model.
+| ![model card template button on the hub ui](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/release-checklist/model-card-template-button.png) | ![model card template on the hub](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/release-checklist/model-card-template.png) |
```

Comment:
Not sure how this will render in a table, given that the left button is very wide. I tried to check [here](https://moon-ci-docs.huggingface.co/docs/hub/pr_2043/en/model-release-checklist), but images are not shown for some reason:

<img width="773" height="211" alt="Screenshot 2025-11-13 at 11 22 57" src="https://github.com/user-attachments/assets/f8b228e1-93f8-4b73-80eb-5beae560b538" />